src/resources/postcss/tickets-admin/attendees/_table.pcss

```diff
@@ -195,13 +201,19 @@
 		display: flex;
 		justify-content: space-between;
 		margin-top: var(--tec-spacer-0);
-		padding: var(--tec-spacer-1) 0 0 var(--tec-spacer-5);
+		padding: var(--tec-spacer-1) 0 0 var(--tec-spacer-5) !important;
```

Comment:
Trust me. I try to avoid using `!important` as much as possible. Because of the CSS that is being used for the `Orders` page (which is a similar layout), I'm forced to do it this way. Good comment, though.